Gemfile

```diff
@@ -30,6 +30,7 @@ when '2.1.6'
 else
   # https://github.com/cerner/cerner_splunk/issues/142
   chef_version = '= 12.18.31'
+  gem 'molinillo', '~> 0.5.0'
```

Comment:
My tests were failing because build_essentials cookbook was not getting installed by berks. I had run into the same issue before while working on an internal project where I found that the transitive dependencies won't get installed when we have 
_molinillo_ >0.6. Hence locked it to 0.5